datacite-doi-commons/src/test/java/no/unit/nva/datacite/DataciteTransformServiceTest.java

```diff
@@ -0,0 +1,65 @@
+package no.unit.nva.datacite;
+
+import java.net.URI;
+import java.time.Instant;
+import java.util.List;
+import no.unit.nva.publication.doi.dto.Contributor;
+import no.unit.nva.publication.doi.dto.DoiRequest;
+import no.unit.nva.publication.doi.dto.DoiRequestStatus;
+import no.unit.nva.publication.doi.dto.Publication;
+import no.unit.nva.publication.doi.dto.Publication.Builder;
+import no.unit.nva.publication.doi.dto.PublicationDate;
+import no.unit.nva.publication.doi.dto.PublicationStatus;
+import no.unit.nva.publication.doi.dto.PublicationType;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class DataciteTransformServiceTest {
+
+    private static final String EXAMPLE_ID = "https://example.net/nva/publicationIdentifier";
+    private static final String EXAMPLE_DOI_ID = "http://doi.org/11541.2/124530";
+    private static final String EXAMPLE_INSTITUTION_OWNER = "???";
+    private static final String EXAMPLE_TITLE = "The Matrix";
+    private static final String EXAMPLE_CONTRIBUTOR_ID = "https://example.net/contributor/id/4000";
+    private static final String EXAMPLE_CONTRIBUTOR_NAME = "Brinx";
+    private static final String EXAMPLE_CONTRIBUTOR_ARPID = "989114";
+    public static final Publication INVALID_PUBLICATION = null;
+
+    @Test
+    public void dataciteTransformerServiceReturnsXmlStringFromPublicationDto() {
+        DataciteTransformService service = new DataciteTransformService();
+        Publication publication = createBuilderWithAllFieldsSet().build();
```

Comment:
Do we want to use the test generator here? If not we might as well delete the code ..

Reply:
Using test data generator now.